ci_framework/roles/devscripts/vars/main.yml

```diff
@@ -47,7 +47,7 @@ cifmw_devscripts_config_defaults:
   master_disk: 100
   master_vcpu: 10
   num_workers: 0
-  num_extra_workers: 3
+  num_extra_workers: 0
```

Comment:
Can you remind why this change?

Reply:
The idea is to use `libvirt_manager` for creating the OSP computes and use `devscripts` for deploying only the OCP control plane. 

Setting this value to `0` prevents `dev-scripts` from creating additional nodes that could be used for compute. Hence changing the default behavior.

Reply:
thanks for the explanation